packages/glibc/pubspec.yaml

```diff
@@ -3,10 +3,12 @@ description: "glibc FFI bindings"
 publish_to: "none"
 
 environment:
-  sdk: ">=2.15.0-0 <3.0.0"
+  sdk: '>=2.17.0 <3.0.0'
 
 dependencies:
+  collection: ^1.16.0
   ffi: ^1.1.2
+  
```

Comment:
nit-pick: could you clean up this, please? p.s. if you use vs code, the "dart add dependency" command helps ;)

Reply:
can you be more specific ?

Reply:
this PR adds a new extra line with two spaces. i was asking if you could clean up the unrelated whitespace change.

Reply:
okay

Reply:
the extra spaces re-appeared a few lines above?

https://github.com/canonical/ubuntu-desktop-installer/pull/899/commits/413ff430cd2370b0b62484530a838568b8cd3171